web/src/stream_settings_ui.js

```diff
@@ -488,6 +488,14 @@ export function redraw_left_panel(left_panel_params = get_left_panel_params()) {
     }
     update_empty_left_panel_message();
 
```

Comment:
Can you add a comment explaining the if block?

Reply:
added class in span `class="settings-empty-option-text"`